fog/view/enclave/impl/src/oblivious_utils.rs

```diff
@@ -569,14 +562,15 @@ mod tests {
     }
 
     #[test]
-    fn collate_shard_query_responses_ciphertext_is_client_ciphertext_length_panics() {
+    fn collate_shard_query_responses_ciphertext_is_greater_than_client_ciphertext_length_panics() {
         let client_search_keys: Vec<Vec<u8>> = (0..10).map(|num| vec![num; 10]).collect();
         let shard_tx_out_search_results: Vec<TxOutSearchResult> = client_search_keys
             .iter()
             .map(|search_key| TxOutSearchResult {
                 search_key: search_key.clone(),
                 result_code: TxOutSearchResultCode::NotFound as u32,
-                ciphertext: vec![0u8; CLIENT_CIPHERTEXT_LENGTH],
+                ciphertext: vec![0u8; CLIENT_CIPHERTEXT_LENGTH + 1],
+                payload_length: (CLIENT_CIPHERTEXT_LENGTH + 1) as u32,
             })
             .collect();
 
```

Comment:
Cleanup for a future PR: https://doc.rust-lang.org/book/ch11-01-writing-tests.html#checking-for-panics-with-should_panic